src/ui.form-input/index.vue

```diff
@@ -12,16 +12,21 @@
   >
     <label :for="id" v-bind="blockAttrs">
       <span
-        v-if="hasSlotContent($slots['left']) || iconLeft"
+        v-if="hasSlotContent($slots['icon-left']) || iconLeft"
         ref="leftSlotWrapper"
-        v-bind="leftSlotAttrs"
+        v-bind="leftIconSlotAttrs"
       >
         <!-- @slot Use it to add icon before text. -->
-        <slot name="left">
+        <slot name="icon-left">
           <UIcon v-if="iconLeft" :name="iconLeft" />
```

Comment:
what about icon size related to input size?

And add props to slot also (same as in UButton).